components/membership/registration-impl/src/integrationTest/kotlin/net/corda/membership/impl/registration/MemberRegistrationIntegrationTest.kt

```diff
@@ -327,6 +342,43 @@ class MemberRegistrationIntegrationTest {
         }
     }
 
+    private fun createMemberList(memberContext: Map<String, String>): List<MemberInfo> {
+        val mgmName = MemberX500Name("Corda MGM", "London", "GB")
+        val mgmId = HoldingIdentity(mgmName, groupId).shortHash.value
+        val ecdhKey = cryptoOpsClient.generateKeyPair(
+            mgmId,
+            CryptoConsts.Categories.PRE_AUTH,
+            mgmId + "ecdh",
+            ECDSA_SECP256R1_CODE_NAME
+        )
+        return listOf(
+            memberInfoFactory.create(
+                sortedMapOf(
+                    PARTY_NAME to mgmName.toString(),
+                    GROUP_ID to groupId,
+                    "corda.endpoints.0.connectionURL" to "localhost:1081",
+                    "corda.endpoints.0.protocolVersion" to "1",
```

Comment:
Could be good to use the constants we have defined for all properties.
```suggestion
                    URL_KEY.format(0) to "localhost:1081",
                    PROTOCOL_VERSION.format(0) to "1",
```

Reply:
addressed in [8ad43b7](https://github.com/corda/corda-runtime-os/pull/3379/commits/8ad43b71841d3ef6e34146370d0c3f7d6fd44645)